include/functions.h

```diff
@@ -1972,12 +1972,10 @@ s32 func_80128640(PlayState* play, Player* player, Gfx* dlist);
 void func_80128B74(PlayState* play, Player* player, s32 limbIndex);
 void func_80128BD0(PlayState* play, s32 limbIndex, Gfx** dList1, Gfx** dList2, Vec3s* rot, Actor* actor);
 
-void PreNMI_Stop(PreNMIContext* prenmiCtx);
-void PreNMI_Update(PreNMIContext* prenmiCtx);
-void PreNMI_Draw(PreNMIContext* prenmiCtx);
-void PreNMI_Main(PreNMIContext* prenmiCtx);
-void PreNMI_Destroy(PreNMIContext* prenmiCtx);
-void PreNMI_Init(PreNMIContext* prenmiCtx);
+void PreNMI_Main(GameState* thisx);
+void PreNMI_Destroy(GameState* thisx);
+void PreNMI_Init(GameState* thisx);
```

Comment:
If you're doing this, you could move them to separate headers, since only very few files need them.